cmake/Modules/FindGLES.cmake: update GLES version

Switch from GLESv1 to GLESv2
---
 cmake/Modules/FindGLES.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Modules/FindGLES.cmake b/cmake/Modules/FindGLES.cmake
index f34f2cd..aef1166 100644
--- a/cmake/Modules/FindGLES.cmake
+++ b/cmake/Modules/FindGLES.cmake
@@ -8,7 +8,7 @@
 #

 find_path(GLES_INCLUDE_DIR GLES/gl.h)
-find_library(GLES_LIBRARY NAMES GLESv1_CM)
+find_library(GLES_LIBRARY NAMES GLESv2)

 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(GLES DEFAULT_MSG GLES_LIBRARY GLES_INCLUDE_DIR)
-- 
2.10.2
